src/main/java/gov/nist/csd/pm/pap/policies/SuperPolicy.java

```diff
@@ -119,13 +119,13 @@ public void configure(Graph graph) throws PMException {
         // check super o is assigned to super oa
         children = graph.getChildren(superOA.getName());
         if(!children.contains(superPolicyClassRep.getName())) {
-            graph.assign(superPolicyClassRep.getName(), superOA.getName());
+            graph.assign(superOA.getName(), superPolicyClassRep.getName());
```

Comment:
Was there an error that led to this change? I think it should be the original assignment rep -> super_oa